src/tests/components/ForecastSummaries.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummaries from "../../components/ForecastSummaries";
+
+describe("ForecastSummaries", () => {
+  const validProps = {
+    forecasts: [
+      {
+        date: 1111111,
+        description: "Stub description1",
+        icon: 511,
+        temperature: {
+          min: 11,
+          max: 22,
+        },
+      },
+      {
+        date: 2222222,
+        description: "Stub description2",
+        icon: 602,
+        temperature: {
+          min: 22,
+          max: 33,
+        },
+      },
+    ],
+    onForecastSelect: () => {},
```

Comment:
Mock functions should be used for valid props here, ie `onForecastSelect: jest.fn()`